src/email_service/terraform/sqs.tf

```diff
@@ -28,3 +28,35 @@ module "send_email_sqs" {
     maxReceiveCount = 2
   }
 }
+
+
+module "auto_resumer_sqs" {
+  source  = "terraform-aws-modules/sqs/aws"
+  version = "4.2.0"
+
+  name                       = "${var.environment}-auto-resumer-sqs"
+  visibility_timeout_seconds = 3600 # Second, make sure it is larger than the Lambda timeout
```

Comment:
After adjusting the auto-resume Lambda timeout, remember to update the visibility_timeout_seconds to better align with the expected execution time of this function.